developers/weaviate/installation/kubernetes.md

```diff
@@ -5,29 +5,26 @@ image: og/docs/installation.jpg
 # tags: ['installation', 'Kubernetes']
 ---
 
-
-:::note Important: Set the correct Weaviate version
-Make sure to set your desired Weaviate version.
-
-This can be done through either explicitly setting it as part of the `values.yaml` or through overwriting the default as outlined in the [deployment step](#deploy-install-the-helm-chart) below.
-:::
-
 :::tip End-to-end guide
-If you are looking for a complete end-to-end tutorial on the topic, see the Weaviate Academy course, [Weaviate on Kubernetes](../../academy/deployment/k8s/index.md). The course is an end-to-end tutorial on how to use MiniKube to deploy Weaviate on Kubernetes.
+For a tutorial on how to use MiniKube to deploy Weaviate on Kubernetes, see the Weaviate Academy course, [Weaviate on Kubernetes](../../academy/deployment/k8s/index.md).
```

Comment:
I would change `MiniKube` to just `Minikube`

Reply:
Updated